DXRando/DeusEx/Classes/OnceOnlyTrigger.uc

```diff
@@ -0,0 +1,23 @@
+//For all those spots where you just don't want it to be possible to trigger something more than once...
+//Stick one of these in the middle.
+class OnceOnlyTrigger extends Trigger;
```

Comment:
I guess we tested this works already? a little surprised we didn't need to check `bDeleteMe` before triggering